src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big__input {
+  padding-left: 62px;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  font-size: 16px;
+}
+
+.small__input {
+  padding-left: 33px;
+  padding-top: 11px;
+  padding-bottom: 10px;
+  font-size: 14px;
+}
```

Comment:
write code according to point 2 of the checklist for the next tasks https://mateacademy.slack.com/archives/C02M6400QF6/p1637221010303800